src/render.spec.ts

```diff
@@ -0,0 +1,54 @@
+import expect from "expect";
+import { getRenderedGameState } from "./render";
+import { GameState, CellValue } from "./gamestate";
+
+function get2RowsGameStateAllCellsEmpty(): GameState {
+  return {
+    board: [
+      [{ value: CellValue.Empty }, { value: CellValue.Empty }],
+      [{ value: CellValue.Empty }, { value: CellValue.Empty }],
+    ],
+  };
+}
+
+function get2RowsGameStateAllCellsBlack(): GameState {
+  return {
+    board: [
+      [{ value: CellValue.Black }, { value: CellValue.Black }],
+      [{ value: CellValue.Black }, { value: CellValue.Black }],
+    ],
+  };
+}
+
+function get2RowsGameStateMixedCellsValues(): GameState {
+  return {
+    board: [
+      [{ value: CellValue.Empty }, { value: CellValue.Black }],
+      [{ value: CellValue.Empty }, { value: CellValue.Black }],
+    ],
+  };
+}
+
+describe("Get 2-rows board with all cells empty must return it", () => {
```

Comment:
We usually wite one "describe" per main function, and many "it" inside for each case